docs/reference/cluster/tasks.asciidoc

```diff
@@ -141,8 +141,8 @@ If the parent isn't found, the API does not return a 404.
 ===== Get more information about tasks
 
 You can also use the `detailed` request parameter to get more information about
-the running tasks. This is useful for telling one task from another but is more
-costly to execute. For example, fetching all searches using the `detailed`
+the running tasks. This is useful for distinguish one task from another task but
+is more costly to execute. For example, fetching all searches using the `detailed`
```

Comment:
How about this?

```suggestion
the running tasks. This is useful to distinguish tasks from each other but
is more costly to execute. For example, fetching all searches using the `detailed`
```

Reply:
Looks better thanks @DaveCTurner Committed suggested change.